### ISSUE TITLE
Admin tab crashes as soon as it is opened

### ISSUE THREAD

**Issue by ggiesen** [Eugeny/tabby#4743]
After upgrading from 1.0.146 to 1.0.159, admin tabs no longer hang with a blank tab (as in #4601), but now they open and disappear immediately.

Running on Windows 11 Pro

**Comment by ggiesen** [comment]
Confirmed this is also an issue on a Windows 10 machine

**Comment by Gpx45** [comment]
Confirmed this as well. I've upgraded and no matter what terminal I set to open as Administrator. It will close out the window immediately after launching from profile.

**Comment by Pandemonium1986** [comment]
Same issue here. 
With a little bit context. I am not a "super admin" and U.A.C is activated on my windows 10 env.
Tabby:  1.0.168
Windows10: 20H2

When I open the Javascript console I have the following error when I try a right click new admin tab

```shell
GET file:///C:/Users/MYUSER/AppData/Local/Programs/Tabby/resources/app.asar/dist/undefined net::ERR_FILE_NOT_FOUND
C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-core\dist\index.js:17440 [local] Destroying
C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5736 Could not read working directory: Error: Failed to read PROCESS_BASIC_INFORMATION.PebBaseAddress
    at getWorkingDirectoryFromPID (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\app.asar\node_modules\native-process-working-directory\index.js:25)
    at Session.<anonymous> (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5733)
    at Generator.next (<anonymous>)
    at C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5250
    at new ZoneAwarePromise (zone.js:1387)
    at __awaiter (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5246)
    at Session.getWorkingDirectory (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5724)
    at Session.<anonymous> (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5609)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\MYUSER\AppData\Local\Programs\Tabby\resources\builtin-plugins\tabby-local\dist\index.js:5247)
    at ZoneDelegate.invoke (zone.js:372)
    at Object.onInvoke (core.js:28666)
    at ZoneDelegate.invoke (zone.js:371)
    at Zone.run (zone.js:134)
    at zone.js:1276
    at ZoneDelegate.invokeTask (zone.js:406)
    at Object.onInvokeTask (core.js:28653)
    at ZoneDelegate.invokeTask (zone.js:405)
    at Zone.runTask (zone.js:178)
    at drainMicroTaskQueue (zone.js:582)
```

**Comment by devopsmyth** [comment]
Same issue here on Windows 11
Tabby version `1.0.168`

As soon as the tab opens, it immediately closes before I can see anything. Doesn't matter which shell I'm attempting to open and I'm running Tabby with Administrator privileges (**Elevated**).

**Comment by Neptear** [comment]
Tabby version 1.0.169
Same issue on Windows11

**Comment by rmaziarka** [comment]
The same issue from my side, with an identical console script as @Pandemonium1986.

**Comment by lack-of-gravitas** [comment]
Same Issue on Windows 11 21H2 (22000.588) running Tabby v1.0.173
Attempted in Admin Mode and standard

**Comment by Eugeny** [comment]
It's fixed in the nightly build.

**Comment by ggiesen** [comment]
Confirming fixed for me in 1.0.174
